src/cli.rs

```diff
@@ -1456,6 +1465,178 @@ mod tests {
         assert!(display.contains("Will flag with aoflagger"));
         assert!(display.contains("Will not correct geometry"));
     }
+
+    /// Middle channel rounded-down is DC flagged when `flag_dc` is set.
+    #[test]
+    fn test_dc_flagging_odd_channel_count() {
+        let (metafits_path, gpufits_paths) = get_1254670392_avg_paths();
+
+        #[rustfmt::skip]
+        let args = vec![
```

Comment:
Super pedantic, but you may not need `vec!` here; a stack array (i.e. `[]`) might be enough.